openlibrary/plugins/openlibrary/js/service-worker.js

```diff
@@ -48,11 +48,11 @@ registerRoute(
 // covers png cache
 registerRoute(
     new RegExp('https://covers.openlibrary.org/b/.+'),
-    new NetworkFirst({
+    new CacheFirst({
         cacheName: 'covers-cache',
         plugins: [
             new ExpirationPlugin({
-                maxEntries: 15,
+                maxEntries: 150,
```

Comment:
Hmmm I'm not sure we want to increase this one up so much ; would it cause too much storage?

An average -M cover is ~15kb with a small random sample form the front page. 15kb  * 150 = 2.25mb . Ok that's pretty tiny! But maybe we should update the cache to only cache `-M.jpg` covers to avoid caching the `-L` covers (which we likely don't care to cache anyways).

Reply:
Actually we should probably also add `purgeOnQuotaError: true,` like the other sections.

Reply:
Good catch